### PR TITLE
CODE RUB: Action variable for testing exception for retrieve all in Calendar Entry Attachment

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 1]
```diff
@@ -25,9 +25,15 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllCalendarEntryAttachmentsW
                 broker.SelectAllCalendarEntryAttachments())
                     .Throws(sqlException);
 
-            // when . then
-            Assert.Throws<CalendarEntryAttachmentDependencyException>(() =>
-                this.calendarEntryAttachmentService.RetrieveAllCalendarEntryAttachments());
+           
+
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs:29]: Remove this extra line @deepshri7

Comment (hassanhabib) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs:29]: Make sure the build runs successfully

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 2]
```diff
@@ -25,13 +25,13 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllCalendarEntryAttachmentsW
                 broker.SelectAllCalendarEntryAttachments())
                     .Throws(sqlException);
 
-            //when
-            Action retrieveAllCalendarEntryAttachmentsAction = () =>
-                this.calendarEntryAttachmentService.RetrieveAllCalendarEntryAttachments();
+            // when
+            Action retrieveAllCalenderEntryAttachmentAction = () =>
+            this.calendarEntryAttachmentService.RetrieveAllCalendarEntryAttachments();
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs:30]: Need tab here

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 3]
```diff
@@ -25,13 +25,13 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllCalendarEntryAttachmentsW
...
 
             // then
             Assert.Throws<CalendarEntryAttachmentDependencyException>(
-                retrieveAllCalendarEntryAttachmentsAction);
+             retrieveAllCalenderEntryAttachmentAction);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs:34]: Check the alignment here.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**